Bottom: 18854afb6b2fef4d972541319707eef7cabe0229
Top:    0f38d28509b44fda4a5750eb5407bb2ff67c5ba6
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-29 14:15:15 -0700

Refresh of rstream-opt

---

diff --git a/examples/rstream.c b/examples/rstream.c
index 6577fe1..78f3c28 100644
--- a/examples/rstream.c
+++ b/examples/rstream.c
@@ -539,6 +539,7 @@ static int run(void)
 	       "name", "bytes", "xfers", "iters", "total", "time", "Gb/sec", "usec/xfer");
 	if (!custom) {
 		optimization = opt_latency;
+optimization = opt_bandwidth;
 		ret = dst_addr ? client_connect() : server_connect();
 		if (ret)
 			goto free;
@@ -547,11 +548,13 @@ static int run(void)
 			if (test_size[i].option > size_option)
 				continue;
 			init_latency_test(test_size[i].size);
+init_bandwidth_test(test_size[i].size);
 			run_test();
 		}
 		rs_shutdown(rs, SHUT_RDWR);
 		rs_close(rs);
 
+sleep(1);
 		optimization = opt_bandwidth;
 		ret = dst_addr ? client_connect() : server_connect();
 		if (ret)
